litmus-portal/cluster-agents/subscriber/pkg/k8s/operations.go

```diff
@@ -165,17 +163,35 @@ func ClusterRegister(clusterData map[string]string) (bool, error) {
 		"CUSTOM_TLS_CERT":      clusterData["CUSTOM_TLS_CERT"],
 	}
 
-	_, err = clientset.CoreV1().ConfigMaps(AgentNamespace).Update(ctx, &corev1.ConfigMap{
+	_, err = clientset.CoreV1().ConfigMaps(AgentNamespace).Update(context.TODO(), &corev1.ConfigMap{
 		Data: newConfigMapData,
 		ObjectMeta: metav1.ObjectMeta{
-			Name: ExternAgentConfigName,
+			Name: AgentConfigName,
+		},
+	}, metav1.UpdateOptions{})
+	if err != nil {
+		return false, err
+	}
+
+	logrus.Info(AgentConfigName + " has been updated")
+
+	newSecretData := map[string]string{
+		"ACCESS_KEY": clusterData["ACCESS_KEY"],
+		"CLUSTER_ID": clusterData["CLUSTER_ID"],
+	}
+
+	_, err = clientset.CoreV1().Secrets(AgentNamespace).Update(context.TODO(), &corev1.Secret{
```

Comment:
Probably need to add secret permissions to subscriber https://github.com/litmuschaos/litmus/blob/035b14eb916ab56ec3f7793330602a0ebe056dc2/litmus-portal/graphql-server/manifests/cluster/3a_agents_rbac.yaml#L185